### DISCUSSION TITLE
Temp licence required

### DISCUSSION THREAD

**Discussion by edjr76** [datalust/seq-tickets#2244] [Help]
Hello, 
We would like to use a licence to help migrate one of our platforms, therefore we would only require it for a couple of weeks to a month. Is this something that Seq can help with?
Many Thanks,
Ed

**Comment by nblumhardt** [comment]
Hi Ed, thanks for dropping us a line. Seq licensing is only available on a yearly basis at present, sorry.

**Comment by edjr76** [comment]
Hi Nicholas,

I appreciate the response.

It would be incredibly helpful if you could let us have a trial licence for a couple of weeks just to move the systems over. We have been a loyal customer for a few years now, and our budgets have been hit quite hard this year – any help or suggestions would be greatly appreciated.

Many Thanks,

Ed

Edward Robinson

Procurement Manager
CABI
Nosworthy Way
Wallingford
Oxfordshire
OX10 8DE
United Kingdom

Tel: +44 (0) 1491 829 247
Email: ***@***.******@***.***>
Visit us at: www.cabi.org<http://www.cabi.org/>

CABI improves people's lives worldwide by providing
information and applying scientific expertise to solve
problems in agriculture and the environment

**Reply by nblumhardt** [reply]
Hi Ed, thanks for the follow-up; coming through on GitHub it's not clear where you're mailing from :-)

Would it be okay to move this conversation to our support address? If so, please reply via support@datalust.co and we'll help you get a solution in place.

Best regards,
Nick
